src/app/systelab-components/modal/message-popup/message-popup-view.component.html

```diff
@@ -2,7 +2,7 @@
 <systelab-message-with-icon class="slab-flex-1 d-flex pl-3 slab-overflow-container" [message]="parameters.msg" [type]="parameters.type" [icon]="parameters.icon"></systelab-message-with-icon>
 <systelab-dialog-bottom>
     <div class="ml-auto" *ngIf="parameters.buttons.length>0">
-        <button *ngFor="let button of parameters.buttons" type="button" class="btn ml-1 {{button.cssClass}}" (click)="close(button.returnValue)">{{button.title}}</button>
+        <button *ngFor="let button of parameters.buttons" #arrayBtn type="button" class="btn ml-1 {{button.cssClass}}" (click)="close(button.returnValue)">{{button.title}}</button>
```

Comment:
It is very weird to have a template reference in an ngFor, as it is going to be a reference to many elements.

Reply:
In fact my tests indicate that it references the first item in the array, but maybe i'm wrong

Reply:
According to some investigations in Internet:
Dynamic Template Reference is not feasible in Angular